nncf/common/graph/graph.py

```diff
@@ -41,6 +42,10 @@ def node_name(self) -> str:
     def node_type(self) -> str:
         return self.data.get(NNCFGraph.NODE_TYPE_ATTR)
 
+    @property
+    def metatype(self) -> Type[OperatorMetatype]:
+        return self.data.get(NNCFGraph.METATYPE_ATTR)
```

Comment:
How should this be different from `node_type`?

Reply:
Node_type is framework operator type, but metatype by definition is a type for grouping framework operators based on their semantic meaning. In the general case, we can not determine framework operator type by metatype.